Add something about Tencent Cloud (#15298)
kubernetes/website: content/en/docs/setup/production-environment/turnkey/tke.md

> ## Running Kubernetes on Tencent Kubernetes Engine

Reviewer: To follow the pattern of other filenames, this file would be named `tencent.md`

### Tencent Kubernetes Engine

> [Tencent Cloud Tencent Kubernetes Engine (TKE)](https://intl.cloud.tencent.com/product/tke) provides easy-to-use and powerful native Kubernetes container management services, enabling you deploy and manage a Kubernetes cluster in just a few steps. For detailed directions, see [Deploy Tencent Kubernetes Engine](https://intl.cloud.tencent.com/document/product/457/11741).

Reviewer: ```suggestion
 [Tencent Cloud Tencent Kubernetes Engine (TKE)](https://intl.cloud.tencent.com/product/tke) provides native Kubernetes container management services. You can deploy and manage a Kubernetes cluster with TKE in just a few steps. For detailed directions, see [Deploy Tencent Kubernetes Engine](https://intl.cloud.tencent.com/document/product/457/11741).
```

My suggested wording.

> In addition to full compatibility with the native Kubernetes API, [TKE](https://intl.cloud.tencent.com/product/tke) features plugins such as CBS and CLB, helping you build a complete system of container services and functions. For more information about how to use [TKE](https://intl.cloud.tencent.com/product/tke), see the [official documentation of TKE](https://intl.cloud.tencent.com/document/product/457).

Reviewer: ```suggestion
 In addition to full compatibility with the native Kubernetes API,TKE features plugins such as CBS and CLB, helping you build a complete system of container services and functions. For learn more about TKE in its official [TKE documentation](https://intl.cloud.tencent.com/document/product/457).
```
No need for a second hyperlink.

Also, what are CBS and CLB? Should readers recognize those terms?

### Custom Deployment

> The core code of [TKE](https://intl.cloud.tencent.com/product/tke) has been made open source on GitHub, which can be obtained [here](https://github.com/TencentCloud/tencentcloud-cloud-controller-manager/blob/master).

Reviewer: ```suggestion
The core of Tencent Kubernetes Engine is open source and available [on GitHub](https://github.com/TencentCloud/tencentcloud-cloud-controller-manager/).
```

> When using [TKE](https://intl.cloud.tencent.com/product/tke) to create a Kubernetes cluster, you can choose managed mode or independent deployment mode. In addition, you can customize the deployment as needed; for example, you can choose an existing CVM instance for cluster creation or enable Kube-proxy IPVS.

Reviewer: ```suggestion
When using TKE to create a Kubernetes cluster, you can choose managed mode or independent deployment mode. In addition, you can customize the deployment as needed; for example, you can choose an existing CVM instance for cluster creation or enable Kube-proxy IPVS.
```
No need for a second hyperlink.

> For more information about usage, see the [official documentation of TKE](https://intl.cloud.tencent.com/document/product/457).

Reviewer: I would drop this line as it's repeating what was stated earlier.
